middleware/logger/tags.go

```diff
@@ -87,13 +88,10 @@ func createTagMap(cfg *Config) map[string]LogFunc {
 			return output.Write(c.Body())
 		},
 		TagBytesReceived: func(output Buffer, c fiber.Ctx, _ *Data, _ string) (int, error) {
-			return appendInt(output, len(c.Request().Body()))
+			return output.WriteString(strconv.Itoa((c.Request().Header.ContentLength())))
```

Comment:
strconv.Itoa is very slow can we find a better solution ?

Reply:
Will be fixing this tonight.

Reply:
@gaby use https://github.com/gofiber/utils/pull/90 ater merge

Reply:
@gaby 
i closed https://github.com/gofiber/utils/pull/90#issuecomment-2242201778

pls use https://pkg.go.dev/strconv#AppendInt

Reply:
Done